sofiadinizms/NFT-onepage
# Feature/new banner

## What I did
1. Updated the banner to the new mockup
2. Adjusted navbar and sidebar to new sections

## How to test
1. Run the project (yarn on the front, npm on the back)
2. Go to your local host 3001
3. Compare the banner with the new mockup
4. Check if it's responsive
5. Take a look at the navbar sections

## Technical debts
1. Routing the navbar
2. Remove the TS errors on the navbar
